packages/components/color-picker/src/use-color-picker.ts

```diff
@@ -459,6 +460,8 @@ export const useColorPicker = (props: UseColorPickerProps) => {
       }
 
       return {
+        "aria-controls": id,
+        role: "textbox",
```

Comment:
Why `role: "textbox"`? I feel like `role: "combobox"`.

Reply:
I've just fixed this